crates/utils/src/tracing/span_ext.rs

```diff
@@ -15,6 +16,13 @@ pub trait ToValue {
     fn to_value(&self) -> Value;
 }
 
+impl<T: ToValue> ToValue for Vec<T> {
+    fn to_value(&self) -> Value {
+        let string_values = self.iter().map(|v| v.to_value().into()).collect::<Vec<StringValue>>();
```

Comment:
Since you're collecting internally, maybe we should use some `Iterator<Item = T>` type?